scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -451,7 +451,7 @@ <h6>Phenotype terms (HPO):</h6>
   {% set symbol_genes_counter = namespace(value=0) %}
   {% set nosymbol_genes_counter = namespace(value=0) %}
   <a
-    href="{{ url_for('panels.panel', panel_id=panel.panel_id) }}"
+    href="{{ url_for('panels.panel', panel_id=panel._id) }}"
```

Comment:
That renaming is such a mess: `_id`, `panel_name`, or `panel_id`? 🙄 Even the view calls several functions and sees if it gets a hit since it doesn't know what kind of name to expect.